client/src/api/UserRegisterApi.js

```diff
@@ -0,0 +1,27 @@
+import USERAPI from "./ApiBase";
+
+const UserRegisterApi = {
+  registerUser: ({
+    firstname,
+    lastname,
+    email,
+    password,
+    password_confirmation,
+  }) => {
+    const options = {
+      method: "POST",
+      url: "/register",
+      data: {
+        firstname: firstname,
```

Comment:
```suggestion
        firstname,
```
if same lang ang name sa variable, it's okay to not assign and retain lang